admin/tabs/roles/HoistRole.ts

```diff
@@ -0,0 +1,34 @@
+export interface HoistRole {
```

Comment:
Didn't see any, but I'll double-check.  Like the idea of admin/Types.ts for any global types, but since this is local to the roles package, maybe would prefer to rename in place